monitoring/install-ks-monitoring.sh

```diff
@@ -21,10 +21,12 @@ ctx="kind-kubeflex"
 ns="ks-monitoring"
 opt="core" # (value: core or wec)
 hub_context="kind-kubeflex"
+env="k8s"
```

Comment:
An OCP cluster _is_ a Kubernetes cluster. In other words, the category of "kubernetes" includes OCP as a subcategory. It would be clearer to use something more specific than "k8s" to contrast with "ocp". Perhaps "base" or "plain"?

Reply:
IMO, the terms "k8s" and "ocp" envs or clusters are very popular and less ambiguous than using very generic terms like "base" or "plain" - what are your thoughts on this?

Reply:
First, let's talk about what `--help` produces. What you have got there is in the syntax of one terse line, but the content has a lot of explanatory material. I think that a better model for the syntax for the output from `--help` would be what you get from, for example, `make --help`:

```console
mspreitz@mjs13 kubestellar % make --help
Usage: make [options] [target] ...
Options:
  -b, -m                      Ignored for compatibility.
  -B, --always-make           Unconditionally make all targets.
  -C DIRECTORY, --directory=DIRECTORY
                              Change to DIRECTORY before doing anything.
  -d                          Print lots of debugging information.
  --debug[=FLAGS]             Print various types of debugging information.
...
```

With this format, you have room for explanatory material.

You can explain that "--env plain" means "plain kubernetes".

Reply:
Ok, I see! I replaced `k8s` with `plain` as you suggested above, and it's the default value.